index.html

```diff
@@ -0,0 +1,143 @@
+            </p>
+        </div>
+        <h2>Latest FIFA News</h2>
+        <div class="headlin-description">
+            <p>
+                FIFA is working on a wide range of topics from improving the women's game and developing football around the world to ensuring better football governance structures.
+            </p>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="title-program">
+            <h3>Main program</h3>
+        </div>
+        <div class="cards">
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/champions.png" alt="champions">
+                </div>
+                <div class="title-card">
+                    <h4>champions</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/laliga.png" alt="laliga">
+                </div>
+                <div class="title-card">
+                    <h4>La liga</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/EnglishPremierLeague.webp" alt="premier">
+                </div>
+                <div class="title-card">
+                    <h4>premier league</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/serie.jpg" alt="serie">
+                </div>
+                <div class="title-card">
+                    <h4>serie A</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please make sure that your program section is similar to the Figma template. I think you could reduce the height, font weight, and font size and also make sure that all the par for every single program card part is exactly the size as defined in the original design. you can check the screenshots below to see the differences. Please fix it in order to stick the design as much as possible :+1: 

#### Original design
![Screenshot from 2022-12-02 17-58-34](https://user-images.githubusercontent.com/6615394/205356829-aa1c49fa-6989-45bb-9a21-171c786513f6.png)

#### Your design
![Screenshot from 2022-12-02 21-32-31](https://user-images.githubusercontent.com/6615394/205391845-88c2ff33-03c6-44d3-bd87-aa380cb38f2a.png)